src/main/java/com/google/sps/data/Journey.java

```diff
@@ -34,10 +34,10 @@
   // waypoint with index b.
   private ArrayList<Pair> restrictions;
   // The starting point for the journey.
-  private Point start;
+  private CourierStop start;
```

Comment:
This theoretically doesn't have to be a CourierStop and can be a plain point. There are no orders to pick up from or deliver to the courier's starting address.

Reply:
Right, but I want to store the `CourierPoint` objects that must be visited in a single array of `CourierStops`. I can change the array to store `Object` instead, but the first version seemed clearer.